net/api/chunked_demo.cc

```diff
@@ -109,9 +109,39 @@ using namespace bricks::net::api;
 using namespace bricks::cerealize;
 using bricks::time::Now;
 using bricks::strings::Printf;
+using bricks::net::HTTPResponseCode;
 
 DEFINE_int32(port, 8181, "The port to serve chunked response on.");
 
+struct LayoutCell {
+  std::string meta_url = "/meta";
+  
+  template <typename A>
+  void serialize(A& ar) {
+    ar(CEREAL_NVP(meta_url));
+  }
+};
+
+struct LayoutItem {
+  std::vector<LayoutItem> row;
+  std::vector<LayoutItem> col;
+  LayoutCell cell;
+
+  template <typename A>
+  void serialize(A& ar) {
```

Comment:
This will not work for de-serialization.

Which is totally fine if you never de-serialize struct ```LayoutItem``` but only serialize it -- but please make sure it does not compile when attempting to de-serealize it then.

Should be a matter of either specifying a non-template type for the parameter (to effectively support ```JSON.stringify()``` type of serialization only), or by inserting a ```static_assert``` checking that the ```A``` is a serialization, not de-serialization medium.